src/map/unit.cpp

```diff
@@ -3421,7 +3421,8 @@ void unit_dataset(struct block_list *bl)
 	ud->steptimer      = INVALID_TIMER;
 	ud->attackabletime =
 	ud->canact_tick    =
-	ud->canmove_tick   = gettick();
+	ud->canmove_tick   =
```

Comment:
```suggestion
	ud->canmove_tick   = gettick();
```

Reply:
If you go this far I would also remove the extra spaces.

Reply:
Track changes / git blame.
If you want something like this, please define a variable above and asign it to all.

Feel free to remove the useless extra spacing.